components/bl-animatedBox-component/component.json

```diff
@@ -0,0 +1,197 @@
+  "description": "The Animated Box component allows the user to create interactive animations.",
+  "showInToolbox": true,
+  "faIcon": "parachute-box",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "select",
+      "name": "type",
+      "label": "Animation Type",
+      "showInSettings": true,
+      "defaultValue": "bounce",
+      "hasLogicHandler": false,
+      "handlerId": "typeLogic",
+      "options": [
+        {
+          "value": "bounce",
+          "label": "bounce"
+        },
+        {
+          "value": "bounceIn",
+          "label": "bounceIn"
+        },
+        {
+          "value": "bounceOut",
+          "label": "bounceOut"
+        },
+        {
+          "value": "flash",
+          "label": "flash"
+        },
+        {
+          "value": "flip",
+          "label": "flip"
+        },
+        {
+          "value": "flipInX",
+          "label": "flipInX"
+        },
+        {
+          "value": "flipInY",
+          "label": "flipInY"
+        },
+        {
+          "value": "heartBeat",
+          "label": "heartBeat"
+        },
+        {
+          "value": "hinge",
+          "label": "hinge"
+        },
+        {
+          "value": "jackInTheBox",
+          "label": "jackInTheBox"
+        },
+        {
+          "value": "jello",
+          "label": "jello"
+        },
+        {
+          "value": "pulse",
+          "label": "pulse"
+        },
+        {
+          "value": "rotateIn",
+          "label": "rotateIn"
+        },
+        {
+          "value": "rubberBand",
+          "label": "rubberBand"
+        },
+        {
+          "value": "shakeX",
+          "label": "shakeX"
+        },
+        {
+          "value": "shakeY",
+          "label": "shakeY"
+        },
+        {
+          "value": "swing",
+          "label": "swing"
+        },
+        {
+          "value": "tada",
+          "label": "tada"
+        },
+        {
+          "value": "wobble",
+          "label": "wobble"
+        },
+        {
+          "value": "zoomIn",
+          "label": "zoomIn"
+        },
+        {
+          "value": "zoomOut",
+          "label": "zoomOut"
+        }
+      ],
+      "settingTooltip": "Select the animation type of the component."
+    },
+    {
+      "type": "number",
+      "name": "duration",
+      "label": "Duration",
+      "settingTooltip": "Duration of the animation, in seconds.",
+      "showInSettings": true,
+      "defaultValue": "1",
+      "hasLogicHandler": false,
+      "handlerId": "durationLogic"
+    },
+    {
+      "type": "number",
+      "name": "delay",
+      "label": "Delay",
+      "showInSettings": true,
+      "defaultValue": "0",
+      "hasLogicHandler": false,
+      "handlerId": "delayLogic",
+      "settingTooltip": "Duration to delay the animation, in seconds."
+    },
+    {
+      "type": "number",
+      "name": "loop",
+      "label": "Loop",
```

Comment:
Im not sure if it's correct to name it Loop.

Basically we use Loop prop as boolean type. There also present some part of boolean logic(we use 0 as infinite look)
One more thing - I couldn't find the case where we need for example 4 animations in the row, but not infinite or one time.

So my offer is make it boolean with variants - 1(loop = false) or infinite(loop = true).